INVALID_SCHEMA_REPLY SERVER system_api#/methods/read_system

### Environment info
- NooBaa Version: **5.1.1-0f82111**
- Platform: **minikube 1.3.1**

### Actual behavior
1. See error below

### Expected behavior
1. No Error

### Steps to reproduce
1. nb install
2. nb backingstore create baba ...
3. nb obc create lala

### More information - Screenshots / Logs / Other output
```
$ kubectl -n noobaa-2 logs noobaa-core-0 noobaa-server | grep INVALID_SCHEMA_REPLY

Sep-4 10:16:36.135 [WebServer/33] [ERROR] core.rpc.rpc_schema:: 
INVALID_SCHEMA_REPLY SERVER system_api#/methods/read_system 

ERRORS: var x = [{
        keyword: 'required',
        dataPath: '.roles[0].account',
        schemaPath: '#/properties/account/required',
        params: {
            missingProperty: 'name'
        },
        message: 'should have required property \'name\'',
        schema: {
            name: {
                '$ref': 'common_api#/definitions/account_name'
            },
            email: {
                '$ref': 'common_api#/definitions/email'
            }
        },
        parentSchema: {
            type: 'object',
            required: ['name', 'email', [length]: 2],
            properties: {
                name: {
                    '$ref': 'common_api#/definitions/account_name'
                },
                email: {
                    '$ref': 'common_api#/definitions/email'
                }
            },
            additionalProperties: false
        },
        data: {}
    }, {
        keyword: 'required',
        dataPath: '.roles[0].account',
        schemaPath: '#/properties/account/required',
        params: {
            missingProperty: 'email'
        },
        message: 'should have required property \'email\'',
        schema: {
            name: {
                '$ref': 'common_api#/definitions/account_name'
            },
            email: {
                '$ref': 'common_api#/definitions/email'
            }
        },
        parentSchema: {
            type: 'object',
            required: ['name', 'email', [length]: 2],
            properties: {
                name: {
                    '$ref': 'common_api#/definitions/account_name'
                },
                email: {
                    '$ref': 'common_api#/definitions/email'
                }
            },
            additionalProperties: false
        },
        data: {}
    },
    [length]: 2
]

REPLY: {
    name: 'noobaa',
    objects: 0,
    roles: [{
            roles: ['admin', [length]: 1],
            account: {}
        }, {
            roles: ['admin', [length]: 1],
            account: {
                name: SENSITIVE - 7594 cb7745b65b69ccb33e86ec4cb189,
                email: SENSITIVE - 7594 cb7745b65b69ccb33e86ec4cb189
            }
        }, {
            roles: ['admin', [length]: 1],
            account: {
                name: SENSITIVE - ccbdf2b517fde47521e1c3a025a60bdc,
                email: SENSITIVE - ccbdf2b517fde47521e1c3a025a60bdc
            }
        }, {
            roles: ['operator', [length]: 1],
            account: {
                name: SENSITIVE - 64 cc919d76dfa2e58ebd823f7004772d,
                email: SENSITIVE - 51e1 d6eab0d3079473554418afe03eaf
            }
        }, {
            roles: ['admin', [length]: 1],
            account: {
                name: SENSITIVE - 64 cc919d76dfa2e58ebd823f7004772d,
                email: SENSITIVE - 60055 b2b369400bc3798ec10235ae237
            }
        },
        [length]: 5
    ],
    buckets: [{
            name: SENSITIVE - bcb7a743c738893faf8bf3f6f1b53fda,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - ebff1601474cbdf59179d34dcbacf14b,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - ebff1601474cbdf59179d34dcbacf14b,
                        spillover: false,
                        disabled: false,
                        mode: 'NOT_ENOUGH_HEALTHY_RESOURCES'
                    },
                    [length]: 1
                ],
                storage: {
                    total: {
                        n: 0,
                        peta: 1
                    },
                    used: 0,
                    used_other: 0,
                    free: 0,
                    reserved: 0,
                    unavailable_free: {
                        n: 0,
                        peta: 1
                    },
                    unavailable_used: 0
                },
                data: {
                    free: 0
                }
            },
            tag: '',
            num_objects: 0,
            writable: false,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 0,
                    free: 0
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 0,
                available_for_upload: 0,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567592010000
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NOT_ENOUGH_HEALTHY_RESOURCES',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'RISKY_TOLERANCE',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        }, {
            name: SENSITIVE - fd426b53e56694a1930ab31331a76a0c,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - 06 f2f0f14721c8bf35e42fe50ca39f5d,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - 06 f2f0f14721c8bf35e42fe50ca39f5d,
                        spillover: false,
                        disabled: false,
                        mode: 'NOT_ENOUGH_HEALTHY_RESOURCES'
                    },
                    [length]: 1
                ],
                storage: {
                    total: {
                        n: 0,
                        peta: 1
                    },
                    used: 0,
                    used_other: 0,
                    free: 0,
                    reserved: 0,
                    unavailable_free: {
                        n: 0,
                        peta: 1
                    },
                    unavailable_used: 0
                },
                data: {
                    free: 0
                }
            },
            tag: '',
            num_objects: 0,
            writable: false,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 0,
                    free: 0
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 0,
                available_for_upload: 0,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567591927346
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NOT_ENOUGH_HEALTHY_RESOURCES',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'RISKY_TOLERANCE',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        }, {
            name: SENSITIVE - e5f8ea8cd843ca4b0593bb44ed15fcc6,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - 2078 bd914850f7829c147591cf5b130c,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - 2078 bd914850f7829c147591cf5b130c,
                        spillover: false,
                        disabled: false,
                        mode: 'NOT_ENOUGH_HEALTHY_RESOURCES'
                    },
                    [length]: 1
                ],
                storage: {
                    total: {
                        n: 0,
                        peta: 1
                    },
                    used: 0,
                    used_other: 0,
                    free: 0,
                    reserved: 0,
                    unavailable_free: {
                        n: 0,
                        peta: 1
                    },
                    unavailable_used: 0
                },
                data: {
                    free: 0
                }
            },
            tag: '',
            num_objects: 0,
            writable: false,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 0,
                    free: 0
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 0,
                available_for_upload: 0,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567590896752
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NOT_ENOUGH_HEALTHY_RESOURCES',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'RISKY_TOLERANCE',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        }, {
            name: SENSITIVE - da7aceaf0ecf7d97e6d996e5ad7ef030,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - 1 c0e69df1ee1d0bf93e3d50e03c7e03a,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - 1 c0e69df1ee1d0bf93e3d50e03c7e03a,
                        spillover: false,
                        disabled: false,
                        mode: 'NOT_ENOUGH_HEALTHY_RESOURCES'
                    },
                    [length]: 1
                ],
                storage: {
                    total: {
                        n: 0,
                        peta: 1
                    },
                    used: 0,
                    used_other: 0,
                    free: 0,
                    reserved: 0,
                    unavailable_free: {
                        n: 0,
                        peta: 1
                    },
                    unavailable_used: 0
                },
                data: {
                    free: 0
                }
            },
            tag: '',
            num_objects: 0,
            writable: false,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 0,
                    free: 0
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 0,
                available_for_upload: 0,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567590381449
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NOT_ENOUGH_HEALTHY_RESOURCES',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'RISKY_TOLERANCE',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        }, {
            name: SENSITIVE - 60842285 fde0c3aed0dd14f587c98ac5,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - 347 edac2d94a96a2af232fc627521139,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - 347 edac2d94a96a2af232fc627521139,
                        spillover: false,
                        disabled: false,
                        mode: 'NOT_ENOUGH_HEALTHY_RESOURCES'
                    },
                    [length]: 1
                ],
                storage: {
                    total: {
                        n: 0,
                        peta: 1
                    },
                    used: 0,
                    used_other: 0,
                    free: 0,
                    reserved: 0,
                    unavailable_free: {
                        n: 0,
                        peta: 1
                    },
                    unavailable_used: 0
                },
                data: {
                    free: 0
                }
            },
            tag: '',
            num_objects: 0,
            writable: false,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 0,
                    free: 0
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 0,
                available_for_upload: 0,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567590349237
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NOT_ENOUGH_HEALTHY_RESOURCES',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'RISKY_TOLERANCE',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        }, {
            name: SENSITIVE - 6000 a515032261ea806c1d14c28dbe5f,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - a2b88fcef793992e617aceb8e4742a22,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - a2b88fcef793992e617aceb8e4742a22,
                        spillover: false,
                        disabled: false,
                        mode: 'NOT_ENOUGH_HEALTHY_RESOURCES'
                    },
                    [length]: 1
                ],
                storage: {
                    total: {
                        n: 0,
                        peta: 1
                    },
                    used: 0,
                    used_other: 0,
                    free: 0,
                    reserved: 0,
                    unavailable_free: {
                        n: 0,
                        peta: 1
                    },
                    unavailable_used: 0
                },
                data: {
                    free: 0
                }
            },
            tag: '',
            num_objects: 0,
            writable: false,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 0,
                    free: 0
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 0,
                available_for_upload: 0,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567589318453
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NOT_ENOUGH_HEALTHY_RESOURCES',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'RISKY_TOLERANCE',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        }, {
            name: SENSITIVE - 0e8 b7657e7f27192d9df73af888fd7ff,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - bb0942ac7872dc5077b26bd29a51f849,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - bb0942ac7872dc5077b26bd29a51f849,
                        spillover: false,
                        disabled: false,
                        mode: 'NOT_ENOUGH_HEALTHY_RESOURCES'
                    },
                    [length]: 1
                ],
                storage: {
                    total: {
                        n: 0,
                        peta: 1
                    },
                    used: 0,
                    used_other: 0,
                    free: 0,
                    reserved: 0,
                    unavailable_free: {
                        n: 0,
                        peta: 1
                    },
                    unavailable_used: 0
                },
                data: {
                    free: 0
                }
            },
            tag: '',
            num_objects: 0,
            writable: false,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 0,
                    free: 0
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 0,
                available_for_upload: 0,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567589318453
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NOT_ENOUGH_HEALTHY_RESOURCES',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'RISKY_TOLERANCE',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        }, {
            name: SENSITIVE - 7e771 ff1dbcac3b19202524c98a8429e,
            namespace: undefined,
            tiering: {
                name: SENSITIVE - 9 ff2dad5860feb221eae946715d8bd0b,
                tiers: [{
                        order: 0,
                        tier: SENSITIVE - 9 ff2dad5860feb221eae946715d8bd0b,
                        spillover: false,
                        disabled: false,
                        mode: 'OPTIMAL'
                    },
                    [length]: 1
                ],
                storage: {
                    total: 5368709120,
                    used: 0,
                    used_other: 73728,
                    free: 5368635392,
                    reserved: 0,
                    unavailable_free: 0,
                    unavailable_used: 0
                },
                data: {
                    free: 5368635392
                }
            },
            tag: '',
            num_objects: 0,
            writable: true,
            spillover: undefined,
            storage: {
                values: {
                    used: 0,
                    used_other: 0,
                    total: 5368635392,
                    free: 5368635392
                },
                last_update: 1567592090524
            },
            data: {
                size: 0,
                size_reduced: 0,
                free: 5368635392,
                available_for_upload: 5368635392,
                last_update: 1567592090524
            },
            usage_by_pool: {
                pools: [
                    [length]: 0
                ],
                last_update: 1567588223338
            },
            quota: undefined,
            stats: undefined,
            stats_by_type: [
                [length]: 0
            ],
            mode: 'NO_RESOURCES_INTERNAL',
            host_tolerance: 0,
            node_tolerance: 0,
            bucket_type: 'REGULAR',
            versioning: 'DISABLED',
            tagging: undefined,
            encryption: undefined,
            bucket_claim: undefined,
            policy_modes: {
                resiliency_status: 'POLICY_PARTIALLY_APPLIED',
                quota_status: 'QUOTA_NOT_SET'
            },
            triggers: [
                [length]: 0
            ]
        },
        [length]: 8
    ],
    namespace_resources: [
        [length]: 0
    ],
    pools: [{
            name: 'baba',
            resource_type: 'CLOUD',
            pool_node_type: 'BLOCK_STORE_S3',
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                free: 0,
                used: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0,
                used_other: 0
            },
            region: undefined,
            create_time: 1567588693000,
            io_stats: {
                read_count: 270,
                write_count: 270,
                read_bytes: 276480,
                write_bytes: 276480
            },
            data_activities: {
                activities: [{
                        reason: 'RESTORING',
                        count: 1,
                        progress: 0.9566433333333333,
                        time: {
                            start: 1567591909111,
                            end: 1567592209111
                        }
                    },
                    [length]: 1
                ],
                host_count: 0
            },
            cloud_info: {
                endpoint: 'https://s3.amazonaws.com',
                endpoint_type: 'AWS',
                target_bucket: 'znoobaa',
                auth_method: 'AWS_V4',
                created_by: SENSITIVE - 60055 b2b369400bc3798ec10235ae237
            },
            undeletable: 'IN_USE',
            mode: 'IO_ERRORS',
            is_managed: true,
            associated_accounts: [
                [length]: 0
            ]
        }, {
            name: 'system-internal-storage-pool-5d6f8033f8e3bc0021b9c134',
            resource_type: 'INTERNAL',
            pool_node_type: 'BLOCK_STORE_MONGO',
            storage: {
                total: 5368709120,
                free: 5368635392,
                used: 0,
                reserved: 0,
                unavailable_free: 0,
                unavailable_used: 0,
                used_other: 73728
            },
            region: undefined,
            create_time: 1567588403000,
            io_stats: {
                read_count: 336,
                write_count: 336,
                read_bytes: 344064,
                write_bytes: 344064
            },
            data_activities: {
                activities: [
                    [length]: 0
                ],
                host_count: 0
            },
            mongo_info: {},
            undeletable: 'IN_USE',
            mode: 'OPTIMAL',
            is_managed: true,
            associated_accounts: [
                [length]: 0
            ]
        },
        [length]: 2
    ],
    tiers: [{
            name: SENSITIVE - ebff1601474cbdf59179d34dcbacf14b,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['baba', [length]: 1],
            mirror_groups: [{
                    name: '5d6f8f03f8e3bc0021b9c181',
                    pools: ['baba', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                used: 0,
                used_other: 0,
                free: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0
            },
            data: {
                free: 0
            }
        }, {
            name: SENSITIVE - 06 f2f0f14721c8bf35e42fe50ca39f5d,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['baba', [length]: 1],
            mirror_groups: [{
                    name: '5d6f8e4ef8e3bc0021b9c177',
                    pools: ['baba', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                used: 0,
                used_other: 0,
                free: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0
            },
            data: {
                free: 0
            }
        }, {
            name: SENSITIVE - 2078 bd914850f7829c147591cf5b130c,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['baba', [length]: 1],
            mirror_groups: [{
                    name: '5d6f8a4bf8e3bc0021b9c16e',
                    pools: ['baba', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                used: 0,
                used_other: 0,
                free: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0
            },
            data: {
                free: 0
            }
        }, {
            name: SENSITIVE - 1 c0e69df1ee1d0bf93e3d50e03c7e03a,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['baba', [length]: 1],
            mirror_groups: [{
                    name: '5d6f882af8e3bc0021b9c165',
                    pools: ['baba', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                used: 0,
                used_other: 0,
                free: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0
            },
            data: {
                free: 0
            }
        }, {
            name: SENSITIVE - 347 edac2d94a96a2af232fc627521139,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['baba', [length]: 1],
            mirror_groups: [{
                    name: '5d6f8828f8e3bc0021b9c15d',
                    pools: ['baba', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                used: 0,
                used_other: 0,
                free: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0
            },
            data: {
                free: 0
            }
        }, {
            name: SENSITIVE - bb0942ac7872dc5077b26bd29a51f849,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['baba', [length]: 1],
            mirror_groups: [{
                    name: '5d6f841df8e3bc0021b9c154',
                    pools: ['baba', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                used: 0,
                used_other: 0,
                free: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0
            },
            data: {
                free: 0
            }
        }, {
            name: SENSITIVE - a2b88fcef793992e617aceb8e4742a22,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['baba', [length]: 1],
            mirror_groups: [{
                    name: '5d6f841cf8e3bc0021b9c14c',
                    pools: ['baba', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: {
                    n: 0,
                    peta: 1
                },
                used: 0,
                used_other: 0,
                free: 0,
                reserved: 0,
                unavailable_free: {
                    n: 0,
                    peta: 1
                },
                unavailable_used: 0
            },
            data: {
                free: 0
            }
        }, {
            name: SENSITIVE - 9 ff2dad5860feb221eae946715d8bd0b,
            chunk_coder_config: {
                digest_type: 'sha384',
                frag_digest_type: 'sha1',
                compress_type: 'snappy',
                cipher_type: 'aes-256-gcm',
                replicas: 1,
                data_frags: 1,
                parity_frags: 0
            },
            data_placement: 'SPREAD',
            attached_pools: ['system-internal-storage-pool-5d6f8033f8e3bc0021b9c134', [length]: 1],
            mirror_groups: [{
                    name: '5d6f8033f8e3bc0021b9c138',
                    pools: ['system-internal-storage-pool-5d6f8033f8e3bc0021b9c134', [length]: 1]
                },
                [length]: 1
            ],
            storage: {
                total: 5368709120,
                used: 0,
                used_other: 73728,
                free: 5368635392,
                reserved: 0,
                unavailable_free: 0,
                unavailable_used: 0
            },
            data: {
                free: 5368635392
            }
        },
        [length]: 8
    ],
    accounts: [{
            name: SENSITIVE - 14 cc973f9ef540dc42b8480d8d01045f,
            email: SENSITIVE - 14 cc973f9ef540dc42b8480d8d01045f,
            access_keys: [{
                    access_key: SENSITIVE - 8379e03386369 bb4bd4c667ca83ed45f,
                    secret_key: SENSITIVE - f40999ff66dee5726a771a86369e04da
                },
                [length]: 1
            ],
            has_login: false,
            has_s3_access: true,
            allowed_buckets: {
                full_permission: false,
                permission_list: [SENSITIVE - bcb7a743c738893faf8bf3f6f1b53fda, [length]: 1]
            },
            default_pool: 'baba',
            can_create_buckets: false,
            systems: [{
                    name: 'noobaa',
                    roles: ['admin', [length]: 1]
                },
                [length]: 1
            ],
            external_connections: {
                count: 0,
                connections: [
                    [length]: 0
                ]
            },
            preferences: {
                ui_theme: 'DARK'
            }
        }, {
            name: SENSITIVE - 7594 cb7745b65b69ccb33e86ec4cb189,
            email: SENSITIVE - 7594 cb7745b65b69ccb33e86ec4cb189,
            access_keys: [{
                    access_key: SENSITIVE - 433 ea29858e37e2a16c6e20d26fafe27,
                    secret_key: SENSITIVE - 7113293115 bc6f4d2c9d580337cb2b21
                },
                [length]: 1
            ],
            has_login: false,
            has_s3_access: true,
            allowed_buckets: {
                full_permission: false,
                permission_list: [SENSITIVE - 60842285 fde0c3aed0dd14f587c98ac5, [length]: 1]
            },
            default_pool: 'baba',
            can_create_buckets: false,
            systems: [{
                    name: 'noobaa',
                    roles: ['admin', [length]: 1]
                },
                [length]: 1
            ],
            external_connections: {
                count: 0,
                connections: [
                    [length]: 0
                ]
            },
            preferences: {
                ui_theme: 'DARK'
            }
        }, {
            name: SENSITIVE - ccbdf2b517fde47521e1c3a025a60bdc,
            email: SENSITIVE - ccbdf2b517fde47521e1c3a025a60bdc,
            access_keys: [{
                    access_key: SENSITIVE - a35acf90db4d370fd3e20b8e1c1d5d53,
                    secret_key: SENSITIVE - 98 f3bad6fe56d100716015218db3df3d
                },
                [length]: 1
            ],
            has_login: false,
            has_s3_access: true,
            allowed_buckets: {
                full_permission: false,
                permission_list: [SENSITIVE - 6000 a515032261ea806c1d14c28dbe5f, [length]: 1]
            },
            default_pool: 'baba',
            can_create_buckets: false,
            systems: [{
                    name: 'noobaa',
                    roles: ['admin', [length]: 1]
                },
                [length]: 1
            ],
            external_connections: {
                count: 0,
                connections: [
                    [length]: 0
                ]
            },
            preferences: {
                ui_theme: 'DARK'
            }
        }, {
            name: SENSITIVE - 64 cc919d76dfa2e58ebd823f7004772d,
            email: SENSITIVE - 60055 b2b369400bc3798ec10235ae237,
            access_keys: [{
                    access_key: SENSITIVE - c111592f19259ca14693d002214c8258,
                    secret_key: SENSITIVE - 568 fcc394ff16d01d5ce479e2dda069e
                },
                [length]: 1
            ],
            has_login: true,
            has_s3_access: true,
            allowed_buckets: {
                full_permission: true
            },
            default_pool: 'baba',
            can_create_buckets: true,
            systems: [{
                    name: 'noobaa',
                    roles: ['admin', [length]: 1]
                },
                [length]: 1
            ],
            external_connections: {
                count: 1,
                connections: [{
                        name: 'baba',
                        endpoint: 'https://s3.amazonaws.com',
                        identity: SENSITIVE - 28 edcafea260c3b7d7d081740a6be88c,
                        auth_method: 'AWS_V4',
                        endpoint_type: 'AWS',
                        usage: [{
                                usage_type: 'CLOUD_RESOURCE',
                                entity: 'baba',
                                external_entity: 'znoobaa'
                            },
                            [length]: 1
                        ]
                    },
                    [length]: 1
                ]
            },
            preferences: {
                ui_theme: 'DARK'
            }
        }, {
            name: SENSITIVE - 64 cc919d76dfa2e58ebd823f7004772d,
            email: SENSITIVE - 51e1 d6eab0d3079473554418afe03eaf,
            access_keys: [{
                    access_key: SENSITIVE - 1 b0be3b8b818ad48bc3a00cc6e4e6cf2,
                    secret_key: SENSITIVE - f87a789d450486c30ef6a64d5b6a4169
                },
                [length]: 1
            ],
            has_login: false,
            has_s3_access: true,
            allowed_buckets: {
                full_permission: true
            },
            default_pool: 'system-internal-storage-pool-5d6f8033f8e3bc0021b9c134',
            can_create_buckets: true,
            systems: [{
                    name: 'noobaa',
                    roles: ['operator', [length]: 1]
                },
                [length]: 1
            ],
            external_connections: {
                count: 0,
                connections: [
                    [length]: 0
                ]
            },
            preferences: {
                ui_theme: 'DARK'
            }
        },
        [length]: 5
    ],
    functions: [
        [length]: 0
    ],
    storage: {
        used: 0,
        total: {
            n: 0,
            peta: 1
        },
        free: 0,
        reserved: 0,
        unavailable_free: {
            n: 0,
            peta: 1
        },
        unavailable_used: 0,
        used_other: 0,
        alloc: 0,
        real: 0
    },
    nodes_storage: {
        used: 0,
        total: 0,
        free: 0,
        reserved: 0,
        unavailable_free: 0,
        unavailable_used: 0,
        used_other: 0,
        alloc: 0,
        real: 0
    },
    nodes: {
        count: 0,
        online: 0,
        by_mode: {},
        storage_count: 0
    },
    hosts: {
        count: 0,
        online: 0,
        by_mode: {},
        by_service: {
            STORAGE: 0,
            GATEWAY: 0
        },
        storage_by_mode: {},
        s3_by_mode: {},
        data_activity_host_count: 0
    },
    owner: {
        name: SENSITIVE - 64 cc919d76dfa2e58ebd823f7004772d,
        email: SENSITIVE - 60055 b2b369400bc3798ec10235ae237,
        access_keys: [{
                access_key: SENSITIVE - c111592f19259ca14693d002214c8258,
                secret_key: SENSITIVE - 568 fcc394ff16d01d5ce479e2dda069e
            },
            [length]: 1
        ],
        has_login: true,
        allowed_ips: undefined,
        has_s3_access: true,
        allowed_buckets: {
            full_permission: true
        },
        default_pool: 'baba',
        can_create_buckets: true,
        systems: [{
                name: 'noobaa',
                roles: ['admin', [length]: 1]
            },
            [length]: 1
        ],
        external_connections: {
            count: 1,
            connections: [
                [length]: 0
            ]
        },
        preferences: {
            ui_theme: 'DARK'
        }
    },
    last_stats_report: 1567590511051,
    maintenance_mode: {
        state: false
    },
    ssl_port: '8443',
    n2n_config: {
        tcp_tls: true,
        tcp_active: true,
        tcp_permanent_passive: {
            min: 60101,
            max: 60600
        },
        udp_dtls: true,
        udp_port: true
    },
    ip_address: '172.17.0.6',
    dns_name: 'noobaa-mgmt.noobaa-2.svc.cluster.local',
    base_address: 'wss://noobaa-mgmt.noobaa-2.svc.cluster.local:8443/',
    phone_home_config: {},
    version: '5.1.1-0f82111',
    node_version: 'v10.14.1',
    debug: {
        level: 0
    },
    system_cap: 20,
    has_ssl_cert: false,
    cluster: {
        master_secret: '26cd3ad6',
        shards: [{
                shardname: 'shard1',
                servers: [{
                        version: '5.1.1-0f82111',
                        hostname: 'noobaa-core-0',
                        secret: '26cd3ad6',
                        addresses: ['172.17.0.6', [length]: 1],
                        status: 'CONNECTED',
                        memory: {
                            total: 4033007616,
                            used: 3752910848,
                            free: 280096768
                        },
                        storage: {
                            total: 17361141760,
                            free: 13296771072
                        },
                        cpus: {
                            count: 4,
                            usage: 0.060085593345147625
                        },
                        location: 'DC1',
                        debug: {
                            level: 0
                        },
                        timezone: 'UTC',
                        dns_servers: [
                            [length]: 0
                        ],
                        time_epoch: 1567592196,
                        services_status: {
                            phonehome_server: {
                                status: 'OPERATIONAL',
                                test_time: 1567592193
                            },
                            cluster_communication: {
                                test_completed: true,
                                results: undefined
                            }
                        }
                    },
                    [length]: 1
                ],
                high_availabilty: false
            },
            [length]: 1
        ],
        min_requirements: {
            ram: 17179869184,
            storage: 128849018880,
            cpu_count: 8
        }
    },
    upgrade: {
        last_upgrade: undefined
    },
    defaults: {
        tiers: {
            data_frags: 4,
            parity_frags: 2,
            replicas: 1,
            failure_tolerance_threshold: 2
        }
    },
    platform_restrictions: [
        [length]: 0
    ]
}
```

--- a/src/server/system_services/stats_aggregator.js
+++ b/src/server/system_services/stats_aggregator.js
@@ -70,33 +70,6 @@ const SYS_STORAGE_DEFAULTS = Object.freeze({
     real: 0,
 });
 
-const PROVIDER_STATS = {
-    'AWS': {
-        logical_size: 0,
-        physical_size: 0,
-    },
-    'AZURE': {
-        logical_size: 0,
-        physical_size: 0,
-    },
-    'S3_COMPATIBLE': {
-        logical_size: 0,
-        physical_size: 0,
-    },
-    'GOOGLE': {
-        logical_size: 0,
-        physical_size: 0,
-    },
-    'KUBERNETES': {
-        logical_size: 0,
-        physical_size: 0,
-    },
-    'OTHERS': {
-        logical_size: 0,
-        physical_size: 0,
-    },
-};
-
 const SINGLE_SYS_DEFAULTS = {
     tiers: 0,
     buckets: 0,
@@ -294,7 +267,7 @@ async function get_partial_accounts_stats(req) {
 
 
 async function get_partial_providers_stats(req) {
-    const provider_stats = _.cloneDeep(PROVIDER_STATS);
+    const provider_stats = {};
     const supported_cloud_types = [
         'AWS',
         'AZURE',
@@ -314,6 +287,12 @@ async function get_partial_providers_stats(req) {
                 if (pool.cloud_pool_info) {
                     type = (supported_cloud_types.includes(pool.cloud_pool_info.endpoint_type)) ?
                         pool.cloud_pool_info.endpoint_type : 'OTHERS';
+                }
+                if (!provider_stats[type]) {
+                    provider_stats[type] = {
+                        logical_size: 0,
+                        physical_size: 0,
+                    };
                 }
                 provider_stats[type].logical_size =
                     size_utils.json_to_bigint(provider_stats[type].logical_size)

--- a/src/server/system_services/system_server.js
+++ b/src/server/system_services/system_server.js
@@ -520,11 +520,12 @@ function read_system(req) {
             objects: objects_sys.count.toJSNumber(),
             roles: _.map(system.roles_by_account, function(roles, account_id) {
                 var account = system_store.data.get_by_id(account_id);
+                if (!account) return;
                 return {
                     roles: roles,
                     account: _.pick(account, 'name', 'email')
                 };
-            }),
+            }).filter(account => !_.isUndefined),
             buckets: _.map(system.buckets_by_name,
                 bucket => {
                     const tiering_pools_status = node_allocator.get_tiering_status(bucket.tiering);

